packages/devtools_app/test/test_infra/scenes/standalone_ui/vs_code_mock_editor.dart

```diff
@@ -74,20 +91,35 @@ class _VsCodeFlutterPanelMockEditorState
                 alignment: Alignment.topCenter,
                 padding: const EdgeInsets.only(top: 60),
                 constraints: const BoxConstraints.expand(width: 48),
-                color: const Color(0xFF333333),
+                color: editorTheme.activityBarBackgroundColor,
                 child: Image.memory(sidebarImageBytes),
               ),
             ),
-            Expanded(child: VsCodeFlutterPanel(api)),
+            Expanded(
+              child: Container(
+                color: editorTheme.sidebarBackgroundColor,
+                child: widget.child ?? const Placeholder(),
+              ),
+            ),
           ],
         ),
         Split(
           axis: Axis.vertical,
           initialFractions: const [0.5, 0.5],
           minSizes: const [200, 200],
+          splitters: [
+            PreferredSize(
+              preferredSize: const Size.fromHeight(1),
+              child: SizedBox(
+                height: 1,
+                child:
+                    ColoredBox(color: editorTheme.editorTerminalSplitterColor),
+              ),
+            ),
+          ],
```

Comment:
nit: do we need to manually specify a splitter here? is the `DefaultSplitter` that is used when `splitters` is null not sufficient?

Reply:
Not really. I was trying to make it look a little more like VS Code (so it gave a better idea of how it'd look) so this was to set the colour and make it narrower. It didn't work particularly well though, so I've removed it for now.